### PR TITLE
feat: added grid component

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

File: list.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                
+                </div>
+
+                <h2>Stacked lists</h2>
+                <p>A responsive image changes its size and adjusts accordingly as the screen size changes</p>
+                <div class="component-container">
+                    <div>
+                        <ul>
+                            <li class="list-stacked">
+                                    <div class="avatar-x-small">
```
Comment (pradipgharge) [inline list.html:126]: just a suggestion, you can use class name like avatar-xs instead of writing complete word

File: toast.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                                d="M 7.71875 6.28125 L 6.28125 7.71875 L 23.5625 25 L 6.28125 42.28125 L 7.71875 43.71875 L 25 26.4375 L 42.28125 43.71875 L 43.71875 42.28125 L 26.4375 25 L 43.71875 7.71875 L 42.28125 6.28125 L 25 23.5625 Z">
+                            </path>
+                        </svg>
+                    
+                    </div>
+                </div>
+
+               
+
+
```
Comment (pradipgharge) [inline toast.html:87]: remove extra spacing here

**Review by pradipgharge — APPROVED** [review]
no major changes from my side, looks good to merge !

File: list.html (hunk 1 above)
Comment (eakansh125) [reply list.html:126]: Sure thing, will keep that in mind going forward

File: toast.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            <div class="main-view-container">
+                <h1>Toast</h1>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <h2>Basic toast</h2>
+                <p>Text fields allow users to enter text into a UI. They typically appear in forms and dialogs.</p>
+                <div class="component-container">
+                    <div class="toast-container red">
+                        <p>This is an alert toast — Click X to dismiss!</p>
+                        <svg class="pointer" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="18" height="18" viewBox="0 0 50 50"
+                            style=" fill:#000000;">
```
Comment (vanshitamoolchandani) [inline toast.html:54]: Change- Avoid inline styling.

File: style.css [hunk 4]
```diff
@@ -635,3 +635,49 @@ main{
...
+    font-size: 1.5rem;
+}
+
+.grid-container-2-cols{
+    grid-template-columns: 1fr 1fr;
+}
+
+.grid-container-3-cols{
+    grid-template-columns: 1fr 1fr 1fr;
+}
```
Comment (rahulk31) [inline style.css:697]: can use repeat() for same fractions.

File: toast.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+                                d="M 7.71875 6.28125 L 6.28125 7.71875 L 23.5625 25 L 6.28125 42.28125 L 7.71875 43.71875 L 25 26.4375 L 42.28125 43.71875 L 43.71875 42.28125 L 26.4375 25 L 43.71875 7.71875 L 42.28125 6.28125 L 25 23.5625 Z">
+                            </path>
+                        </svg>
+                        
+                    </div>
+
+                    <div class="toast-container green">
+                        <p>This is a success toast — Click X to dismiss!</p>
+                        <svg class="pointer" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="18" height="18" viewBox="0 0 50 50"
+                            style=" fill:#000000;">
```
Comment (vanshitamoolchandani) [inline toast.html:65]: same here

File: style.css [hunk 6]
```diff
@@ -53,7 +69,29 @@ body{
...
+.pointer{
+    cursor: pointer;
+}
+
+.background-light-grey{
+    background-color: var(--light-grey);
+}
 
+.position-static{
+    position: static !important;
```
Comment (Naman-Saxena1) [inline style.css:81]: Generally, try avoiding important and use something with more specificity. Until unless it can't be done without !important

**Review by rahulk31 — APPROVED** [review]
Couldn't find flaw in code. Well structured and good use of classes can be seen. Great work. This can be merged. 👍

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

**Review by vanshitamoolchandani — APPROVED** [review]
Just few minor changes, other than that you are good to merge.

File: style.css (hunk 4 above)
Comment (eakansh125) [reply style.css:697]: Sure will keep that in check

File: toast.html (hunk 2 above)
Comment (eakansh125) [reply toast.html:87]: Sure thing

**Comment by eakansh125** [comment]
> Couldn't find flaw in code. Well structured and good use of classes can be seen. Great work. This can be merged. 👍

Thanks much!!

File: toast.html (hunk 3 above)
Comment (eakansh125) [reply toast.html:54]: Actually it's a copied svg, but yeah I got your point. Will keep that in check. Thank you!

File: style.css (hunk 6 above)
Comment (eakansh125) [reply style.css:81]: Sure thing, will keep that in check

**Pull request merged**